### DISCUSSION TITLE
Is there a function that has the same functionality as 'set' in @nuxtjs/composition-api?

### DISCUSSION THREAD

**Discussion by atlansien** [nuxt/nuxt#20430] [Questions]
```
import { set } from '@nuxtjs/composition-api';
```
node_modules/@vue/composition-api/dist/vue-composition-api.d.ts
```
/**
 * Set a property on an object. Adds the new property, triggers change
 * notification and intercept it's subsequent access if the property doesn't
 * already exist.
 */
declare function set<T>(target: AnyObject, key: any, val: T): T;
```
If anyone knows of a function in Nuxt3 that has the same functionality as this one, please let me know.

**Comment by peterroe** [comment]
Since `vue3` implements `reactive` with `proxy`, I think you don't need `set` in `nuxt3`, could you describe your scene ？👀
